### PR TITLE
Feature/add status component

### LINKED ISSUE #67: Add component status connetion

### PR DESCRIPTION
**Pull request by VladKalachev**
- add component status
- add scss module for component
Close #67

### CODE REVIEW

File: packages/cli-ui/src/components/ConnectionStatus/ConnectionStatus.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  const [connected, setConnected] = useState(true)
+  const [status, setStatus] = useState('show')
+
+  useEffect(() => {
+    socket.on('connect', () => {
+        setStatus('show')
+        setConnected(true)
+        setTimeout(() => {
+            setStatus('hidden')
+        }, 700)
```
Comment (KovaSs) [inline packages/cli-ui/src/components/ConnectionStatus/ConnectionStatus.tsx:17]: Don't forget to use batch
```suggestion
         batch(() => {
            setStatus('show')
            setConnected(true)
            setTimeout(() => {
                setStatus('hidden')
            }, 700)
        })
```

File: packages/cli-ui/src/components/ConnectionStatus/ConnectionStatus.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        socket.on('disconnect', function() {
+            console.log('disconnect')
+            setStatus('show')
```
Comment (KovaSs) [inline packages/cli-ui/src/components/ConnectionStatus/ConnectionStatus.tsx:20]: And this

File: packages/cli-ui/src/components/FileManager/FileManager.tsx [hunk 3]
```diff
@@ -17,10 +17,10 @@ export default function FileManager () {
...
-      type: "GET_FOLDERS",  
-      url: `/${url.join('/')}`,  
-      hidden: false 
+    socket.send({
+      type: 'GET_FOLDERS',
+      url: `/${url.join('/')}`,
+      hidden: false
     })
 
     socket.on('folders', (res) => {
```
Comment (KovaSs) [inline packages/cli-ui/src/components/FileManager/FileManager.tsx:26]: Put setLoading(true) into batch

File: packages/cli-ui/src/components/index.ts [hunk 4]
```diff
@@ -12,3 +12,4 @@ export { default as CheckBoxTheme } from './CheckBoxTheme'
 export { default as Empty } from './Empty'
 export { default as FileManager } from './FileManager'
 export { default as Modal } from './Modal'
+export { default as ConnectionStatus } from './ConnectionStatus'
```
Comment (KovaSs) [inline packages/cli-ui/src/components/index.ts:15]: Plz new line)

File: packages/cli-ui/src/components/ConnectionStatus/style.module.scss [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+	.connected{
+		background: #61dafb;
+		padding: 16px;
+	}
+
+	.disconnected{
+		background: red;
+		padding: 16px;
+	}
+}
```
Comment (KovaSs) [inline packages/cli-ui/src/components/ConnectionStatus/style.module.scss:20]: Plz new line)

File: packages/cli-ui/src/components/ConnectionStatus/ConnectionStatus.tsx [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+      socket.off('connect')
+      socket.off('disconnect')
+      setConnected(false)
+    }
+  }, [])
+
+  return (
+    <div className={`${css.content} ${status === 'show' ? css.show : css.hidden}`}>
+      {connected ? (<div className={css.connected}>
+        Connected
```
Comment (KovaSs) [inline packages/cli-ui/src/components/ConnectionStatus/ConnectionStatus.tsx:38]: Maybe localized?

File: packages/cli-ui/src/components/ConnectionStatus/ConnectionStatus.tsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+      </div>) : (<div className={css.disconnected}>
+        Disconnected
```
Comment (KovaSs) [inline packages/cli-ui/src/components/ConnectionStatus/ConnectionStatus.tsx:40]: Maybe localized?

Comment (VladKalachev) [reply packages/cli-ui/src/components/ConnectionStatus/ConnectionStatus.tsx]: add localize

**Pull request merged**